### DISCUSSION TITLE
All LEDs are always on

### DISCUSSION THREAD

**Discussion by oilcan-productions** [thehighnibble/thehighnibble.github.io#140] [IMSAI8080esp replica]
I moved my IMSAI 8080 the other day from one room to another. After plugging it back in, some or all LEDs are on regardless of Power Switch ON or OFF position. When I turn the Power Switch OFF, the LEDs all flash quickly then return to a random ON state sometimes all sometimes only a few. When powered on all LEDs are on. I checked for stuck switches etc. Seems all good. 
System works otherwise fine though.

Anything I can do more to troubleshoot?

**Comment by nbreeden2** [comment]
When you say, "System works otherwise fine though." do you mean you can boot into CP/M and the WebUI is working?

With the system in the state were some LEDs are on / Off.
Lift RESET and hold it up. All of the address, data and programmed LEDs should light.
Press down EXT CLR and release it. All of the address, data and programmed LEDs should flash.
When the system is booted and running CP/M is there activity on the LEDs.
When you press STOP the WAIT LED should lite up. When you press RUN the RUN LED should light up. Is this happeneing?

**Comment by oilcan-productions** [comment]
Yes, I can boot to CP/M and web ux works. None of the switches except the power switch make a difference.


Sent from Surface Duo
________________________________

**Comment by oilcan-productions** [comment]
Problem solved. When moving one of the solder points of the ESP socket came lose. I resoldered the socket and now the front panel is working again.
